cylc/flow/network/schema.py

```diff
@@ -985,35 +987,36 @@ def resolve_task_id(root, info, **args):
 
 class Prerequisite(ObjectType):
     class Meta:
-        description = """Task prerequisite."""
+        description = 'Task prerequisite.'
     expression = String()
     conditions = graphene.List(
         Condition,
-        description="""Condition monomers of a task prerequisites.""")
+        description='Condition monomers of a task prerequisites.'
```

Comment:
```suggestion
        description='Condition monomers of a task prerequisite.'
```

Reply:
"Condition monomers" - that's a good one!

Reply:
Yeah who's the wordsmith responsible for that gem?